integrations/java/iceberg-1.2/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseTableOperations.java

```diff
@@ -301,8 +301,6 @@ static Mono<GetTableResponseBody> handleCreateUpdateHttpError(
               casted, casted.getStatusCode().value() + " , " + casted.getResponseBodyAsString()));
     } else if (e instanceof WebClientResponseException) {
       return Mono.error(new WebClientResponseWithMessageException((WebClientResponseException) e));
-    } else if (e instanceof WebClientRequestException) {
-      return Mono.error(new WebClientRequestWithMessageException((WebClientRequestException) e));
     } else {
       /**
        * This serves as a catch-all for any unexpected exceptions that could occur during doCommit,
```

Comment:
Can we also a comment to explicitly mention webClientRequestException here?

Reply:
added